great_expectations/render/renderer/content_block/table_content_block.py

```diff
@@ -63,9 +63,9 @@ def expect_column_mean_to_be_between(cls, ge_object):
     @classmethod
     def expect_column_values_to_not_be_null(cls, ge_object):
         return [
-            ["Missing (n)", ge_object["result"]["unexpected_count"] if ge_object["result"].get("unexpected_count") else "--"],
```

Comment:
Replacing the "get" check with the "in" check makes sure that 0's are not interpreted as None.